From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@ShawnJackson__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

v-viinde committed:
USER STORY 1032046 SaaS App Tutorial: SumoLogic Tutorial Update

v-vikum2 committed:
USER STORY 1032719 SaaS App Tutorial: Veritas Enterprise Vault.cloud SSO Update

heyyjudes committed:
Adding Change Feed Processor Library Inital Commit

Adding updates to Change feed article

heyyjudes committed:
Fixing Formatting Issues

heyyjudes committed:
Improving Acrolinx score spelling and grammar

heyyjudes committed:
Fixing container vs collection terminology

heyyjudes committed:
Changing to new change feed overview image

heyyjudes committed:
Removing duplicates and fixing links

heyyjudes committed:
adding in page link

jpallavi committed:
Update backup-azure-vms-encryption.md

New section for giving permissions to Azure Backup.

jpallavi committed:
Add files via upload

jpallavi committed:
Add files via upload

jpallavi committed:
Delete encrypted-vm-warning-message.png

jpallavi committed:
Add files via upload

jpallavi committed:
Delete search-key-vault.png

jpallavi committed:
Add files via upload

jpallavi committed:
Update backup-azure-vms-encryption.md

jpallavi committed:
Update backup-azure-vms-encryption.md

jpallavi committed:
Add files via upload

jpallavi committed:
Update backup-azure-vms-encryption.md

jpallavi committed:
Add files via upload

jpallavi committed:
Add files via upload

jpallavi committed:
Update backup-azure-vms-encryption.md

jpallavi committed:
Update backup-azure-vms-encryption.md

jpallavi committed:
Update backup-azure-vms-encryption.md

mimig1 committed:
Editorial changes

Commit:
application gateway : web apps

heyyjudes committed:
Merge pull request #1 from mimig1/patch-118

Editorial changes

heyyjudes committed:
Merge branch 'master' into master

jpallavi committed:
Update backup-azure-vms-encryption.md

jpallavi committed:
Update backup-azure-vms-encryption.md

v-viinde committed:
USER STORY 1044431 SaaS App Tutorial: Workstars

v-viinde committed:
Updated

Commit:
edit pass: active-directory-sso-certs

Commit:
app gateway web app updates

Commit:
updates to faq

barclayn committed:
replacing image

barclayn committed:
replacing image and updating the refresh date

Commit:
update to image

LoriMerrick committed:
Update active-directory-sso-certs.md

Made minor changes.

MicrosoftGuyJFlo committed:
SaaS Tutorial Landing Page Updates

Commit:
resized imagE

Commit:
resized imagE

Commit:
Update log-analytics-usage.md

Including details for troubleshooting Azure Diagnostics

mimig1 committed:
Updated change-feed.md to disambiguate between new SDK and old SDKs

-Removed statement about incurring additional costs as the new SDK would incur additional costs with the leased collection. 
-Disambiguated between using the existing SDKs and the new Change Feed Processor library.
-Added links to the ReadDocumentFeedAsync API ref page. 
-Formatted code blocks as csharp.
-Corrected incorrect casing in table title and text.

Commit:
updates to docs based on acrolinx

MicrosoftGuyJFlo committed:
SaaS Tutorial Landing Page Updates

mimig1 committed:
Added a note about change feed only working for documents at this time

MicrosoftGuyJFlo committed:
SaaS Tutorial Landing Page Updates

v-thepet committed:
inserting the code samples links

Commit:
fix usability issues

Commit:
updating PS

JasonWHowell committed:
Update connect-csharp.md

JasonWHowell committed:
Update connect-csharp.md

JasonWHowell committed:
Update connect-csharp.md

JasonWHowell committed:
Update connect-csharp.md

MicrosoftGuyJFlo committed:
SaaS Tutorial Landing Page Updates

Commit:
Merge pull request #18176 from stevestein/SqlDbUpdate7261

fix usability issues

v-thepet committed:
Merge pull request #18178 from HeathL17/hl-serviceprincipal

updating PS

Lisaco88 committed:
Merge pull request #18177 from v-thepet/roadmap4

production bug 71893: part 2 of 3, inserting the code samples links

tfitzmac committed:
added ps parameters

MicrosoftGuyJFlo committed:
SaaS Tutorial Landing Page Updates

Commit:
Merge pull request #18180 from JasonWHowell/patch-5

MySQL: Update connect-csharp.md

Commit:
Merge pull request #18179 from JasonWHowell/patch-2

PostgreSQL: Update connect-csharp.md

CarlRabeler committed:
Update sql-database-export.md

JasonWHowell committed:
Update TOC.yml

MicrosoftGuyJFlo committed:
Update to article per PRMerger find

Commit:
Merge pull request #18182 from tfitzmac/0726policy

added ps parameters

Commit:
Merge pull request #18183 from CarlRabeler/patch-304

Update sql-database-export.md

v-thepet committed:
add the to link

Lisaco88 committed:
Merge pull request #18109 from v-viinde/workstars

USER STORY 1044431 SaaS App Tutorial: Workstars

MicrosoftGuyJFlo committed:
SaaS Tutorial Landing Page Updates

Commit:
Merge pull request #18185 from MicrosoftGuyJFlo/SAPHANARevision

Update to article per PRMerger find

Commit:
Merge pull request #18186 from v-thepet/ugh

production bug: add "the" to link

v-thepet committed:
Merge pull request #18184 from JasonWHowell/patch-1

PostgreSQL: Update TOC.yml

heyyjudes committed:
Merge pull request #2 from mimig1/patch-120

Updated change-feed.md to disambiguate between new SDK and old SDKs

Commit:
Adding deprecation notice for Recommendations API.

mmacy committed:
[aci] fix container group template error

Commit:
Fixing a few grammar and style issues.

Commit:
Merge pull request #18189 from richrundmsft/patch-4

Update log-analytics-usage.md

Commit:
Merge pull request #18191 from mmacy/aci-multi-group-fix

[aci] fix container group template error

cherylmc committed:
update format

Lisaco88 committed:
Merge pull request #16815 from v-viinde/sumologic

USER STORY 1032046 SaaS App Tutorial: SumoLogic Tutorial Update

Commit:
Changed note notation to follow Azure's style

cherylmc committed:
update

cherylmc committed:
update format

cherylmc committed:
update

MicrosoftGuyJFlo committed:
SaaS Tutorial Landing Page Updates

MicrosoftGuyJFlo committed:
SaaS Tutorial Landing Page Updates

bketelsen committed:
Update environment variable

STORAGE_ACCT -> STORAGE_ACCOUNT

curtand committed:
REALLY the last classic portal updates

anzaman committed:
Update expressroute-locations.md

anzaman committed:
Update expressroute-locations-providers.md

Commit:
Merge pull request #18197 from curtand/femila0726

REALLY the last classic portal updates

Commit:
Merge pull request #18198 from anzaman/patch-11

Update expressroute-locations.md

Commit:
Merge pull request #18199 from anzaman/patch-12

Update expressroute-locations-providers.md

vladvino committed:
Clarified policy scoping

Commit:
Merge pull request #18200 from vladvino/patch-10

Clarified policy scoping

GitHubber17 committed:
Merge pull request #16828 from v-vikum2/veritas

USER STORY 1032719 SaaS App Tutorial: Veritas Enterprise Vault.cloud SSO Update

GitHubber17 committed:
Merge pull request #18165 from MicrosoftGuyJFlo/SaaSTutotialListUpdate

SaaS tutotial list update

GitHubber17 committed:
Merge pull request #18193 from cherylmc/erfaqup

update format

anoobbacker committed:
Updating link

Commit:
Merge pull request #18202 from anoobbacker/patch-9

Updating link

barclayn committed:
fixing acrolynx comments

Commit:
corrected grammar issues on overview

jpallavi committed:
Delete backup-service-access-policy.png

jpallavi committed:
Add files via upload

jpallavi committed:
Delete save-access-policy.png

jpallavi committed:
Delete select-key-vault-access-policy.png

jpallavi committed:
Delete select-key-vault.png

jpallavi committed:
Add files via upload

jpallavi committed:
Update backup-azure-vms-encryption.md

jpallavi committed:
Update backup-azure-vms-encryption.md

linda33wj committed:
refresh copy content

Commit:
Merge pull request #18206 from linda33wj/master

refresh copy content

dominicbetts committed:
Add communications patterns

Commit:
Merge pull request #18207 from dominicbetts/cfupdates

Add communications patterns

Commit:
updates per engineering

Commit:
Merge pull request #18212 from Blackmist/fresh2

updates per engineering

AnuragMehrotra committed:
Update tutorial-backup-vms.md

AnuragMehrotra committed:
Update tutorial-backup-vms.md

Commit:
Merge pull request #18215 from AnuragMehrotra/patch-1

Update tutorial-backup-vms.md

mimig1 committed:
Cosmos DB: Added tip about not updating serializer value per customer feedback.

Commit:
Merge pull request #18216 from mimig1/patch-121

Cosmos DB: Added tip about not updating serializer value per customer…

mmacy committed:
[stg] update Azure Files section

vhorne committed:
Acrofixes

mmacy committed:
[stg] update Azure Files section

Lisaco88 committed:
Merge pull request #17946 from JPallavi/patch-24

Update backup-azure-vms-encryption.md

cynthn committed:
Update tutorial-backup-vms.md

Commit:
Merge pull request #18217 from mmacy/stg-intro-file-mount-fix

[stg] update Azure Files section

Lisaco88 committed:
Merge pull request #18027 from georgewallace/appgatewaywebapp

application gateway : web apps

cynthn committed:
Update tutorial-backup-vms.md

vhorne committed:
remove TP3

Commit:
Merge pull request #18220 from cynthn/patch-45

Update tutorial-backup-vms.md

mimig1 committed:
Cosmos DB: Added info about running on a local network.

mimig1 committed:
typo fix

Commit:
Merge pull request #18221 from cynthn/patch-46

Update tutorial-backup-vms.md

Lisaco88 committed:
Merge pull request #17682 from heyyjudes/master

Adding Change Feed Processor Library Inital Commit

v-thepet committed:
Merge pull request #18190 from LuisCabrer/patch-2

Adding deprecation notice for Recommendations API.

Lisaco88 committed:
Merge pull request #18155 from barclayn/azlog-intro

Azlog intro

billmath committed:
updating

YuriDio committed:
updating article

Commit:
Merge pull request #18224 from billmath/quality

updating

YuriDio committed:
updating article - fixing

Commit:
Merge pull request #18223 from mimig1/07272017md

Cosmos DB: Added info about running emulator on a local network.

ShawnJackson committed:
Update active-directory-sso-certs.md

Commit:
Merge pull request #18196 from bketelsen/patch-1

Update environment variable

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

rjagiewich committed:
Merge pull request #18218 from vhorne/fix-dns

Acrofixes

ShawnJackson committed:
Update active-directory-sso-certs.md

v-alje committed:
Merge pull request #18226 from MicrosoftDocs/FromPublicRepo

From public repo

ShawnJackson committed:
Merge pull request #18140 from LoriMerrick/active-directory-sso-certs2

edit pass: active-directory-sso-certs

rjagiewich committed:
Merge pull request #18225 from YuriDio/JulyUpdates_8

updating article
